src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+
+.. index:: gtin
+.. _feed_gtin: 
+
+GTIN
+""""""""""""""""""""""
+
+GTIN, definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 50 chars.
+
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:gtin>44320194113475</ad:gtin>
+======= ==================================
+
+.. index:: itemGroupId
+.. _feed_itemGroupId: 
+
+itemGroupId
+""""""""""""""""""""""
+
+Item Group Id definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 1-50 chars.
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:itemGroupId>BC23456</ad:itemGroupId>
+======= ============================================
+
+.. index:: condition
+.. _feed_condition: 
+
+condition
+""""""""""""""""""""""
+
+Condition definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+Accepted values: *new*, *refurbished*, *used*
+
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:condition>used</ad:condition>
+======= ==================================
+
+.. index:: material
+.. _feed_material: 
+
+material
+""""""""""""""""""""""
+
+Material definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 200 chars.
+  
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:material>Wool</ad:material>
+======= ==================================
+
+.. index:: energyEfficiencyClass
+.. _feed_energyEfficiencyClass: 
+
+energyEfficiencyClass
+""""""""""""""""""""""
+
+Energy Efficiency Class See `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_
+
+Allowed values: *A+++*, *A++*, *A+*, *A++*,	*B*, *C*, *B*, *E*,	*F*, *G*
+
+======= ===========================================================
+Example .. code-block:: html
+        
+         <ad:energyEfficiencyClass>A+</ad:energyEfficiencyClass>
+======= ===========================================================
+
+.. index:: minEnergyEfficiencyClass
+.. _feed_minEnergyEfficiencyClass: 
+
+minEnergyEfficiencyClass
+""""""""""""""""""""""""
+
+Minimal energy efficiency class. Possible values defined in :ref:`feed_energyEfficiencyClass`
+
+======= ================================================================
+Example .. code-block:: html
+        
+         <ad:minEnergyEfficiencyClass>E</ad:minEnergyEfficiencyClass>
+======= ================================================================
+
+.. index:: maxEnergyEfficiencyClass
+.. _feed_maxEnergyEfficiencyClass: 
+
+maxEnergyEfficiencyClass
+""""""""""""""""""""""""
+
+Maximal energy efficiency class. Possible values defined in :ref:`feed_energyEfficiencyClass`
```

Comment:
I think for both min and max we should mention it is used in combination with `energyEfficiencyClass` as per google spec, right now it's confusing to me why I would use it.

Reply:
ok